### DISCUSSION TITLE
Created a Build .wgt file but can't install on a target tv emulator

### DISCUSSION THREAD

**Discussion by HatimPatwa** [pavjacko/renative#1169] [Q&A] [answered]
_Created a Build .wgt file but can't install on a target tv emulator_

Hey guys i just created a **.wgt** file through command **rnv build -p tizen -s release**

But i cant seem to load  and install it into a tizen Emulator to run standalone , below are the steps i tried

1. Drag and drop .wgt file on emulator -> pop up asks to install, clicked install -> shows loader for like a second and nothing happens. cant find the app  installed anywhere

2. shifted the .wgt file to tizen studio but there the studio says there is an issue in configuration when i try to run 

I know .wgt is the final step to get your application to run standalone

please any help or discussion would be appreciated

**Comment by pauliusguzas** [comment, marked as answer]
Closing as issue tracked in ticket
